parlai/tasks/convai2/worlds.py

```diff
@@ -76,10 +82,15 @@ def finalize_episode(self):
             print(f"Your partner was playing the following persona:\n{partner_persona}")
         print("[ Preparing new chat ... ]\n")
 
+    def share(self):
+        shared_data = super().share()
+        shared_data['personas_list'] = self.personas_list
+        return shared_data
+
 
 class SelfChatWorld(SelfChatBaseWorld):
     def init_contexts(self):
-        self.personas_list = load_personas(self.opt)
+        self.personas_list = get_personas(self.opt)
```

Comment:
why not the same sharing in selfchat worlds?

Reply:
I was just trying to limit the scope of this PR. But I'd be very happy to add it to SelfChat worlds as well!